packages/dbgpt-app/src/dbgpt_app/openapi/api_v1/subagent/react_tools.py

```
@@ -0,0 +1,1351 @@
+
+                # Match src="..." or src='...' containing image references
+                fixed_html = re.sub(
+                    r"""(src\s*=\s*["'])"""
+                    r"""([^"']+\.(?:png|jpg|jpeg|gif|svg|webp))"""
+                    r"""(["'])""",
+                    _fix_img_src,
+                    fixed_html,
+                    flags=re.IGNORECASE,
+                )
+        except Exception:
+            pass  # If post-processing fails, use original HTML
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)